remote_exec/remote_exec.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+option go_package = "teleport_exec/remote_exec";
+package remote_exec;
+
+//-----------------------------------------------------------------------------
+message StartCommandRequest { string command = 1; }
+
+//-----------------------------------------------------------------------------
+message CommandStatusRequest { string command_id = 1; }
+
+message CommandStateResponse {
+  string command_id = 1;
+  string command = 2;
+  bool running = 3;
+  optional int32 result_code = 4;
+}
+
+//-----------------------------------------------------------------------------
+message StopCommandRequest { string command_id = 1; }
+
+message StopCommandResponse {
+  string command_id = 1;
+  bool success = 2;
+}
+
+//-----------------------------------------------------------------------------
+message CommandOutputRequest {
+  string command_id = 1;
+
+  // If the output should be streamed until the command stops
+  bool stream = 2;
```

Comment:
You don't need to implement this as a choice, you can always assume stream until the command stops.

Reply:
Great, will do that.